### PR TITLE
feat: add reviewed type pull request event

### PR DESCRIPTION
**Pull request by pd4d10**
| Question         | Response    |
| ---------------- | ----------- |
| Version?         | v1.4.1      |
| Devices tested?  | iPhone 7... |
| Bug fix?         | yes/no      |
| New feature?     | yes/no      |
| Includes tests?  | yes/no      |
| All Tests pass?  | yes/no      |
| Related ticket?  | #...        |

---

## Screenshots

| Before   | After    |
| -------- | -------- |
|before|after

### CODE REVIEW

**Comment by chinesedfan** [comment]
@pd4d10 Nice start. I didn't have a chance to run your codes because of a little busy, but I can share you some comments first.

- Will `reviewed` event contain other states like `request changes`? If so, the green check icon should switch to a red cross icon.
- The check icon seems not in the center, compared with Github's. 
- New introduced words should support i18n. Or someone can do it together in #795.

**Comment by pd4d10** [comment]
Sorry for the late response. `request changes` type added, and icon positions also fixed.

i18n support is not added in this PR because different languages mixed seems weird... Is it better to do it together in #795?

File: src/components/issue-event-list-item.component.js [hunk 1]
```diff
@@ -297,6 +302,42 @@ export class IssueEventListItem extends Component {
             createdAt={event.created_at}
           />
         );
+      case 'reviewed':
+        switch (event.state) {
+          case 'approve':
```
Comment (chinesedfan) [inline src/components/issue-event-list-item.component.js]: `approved`.

**Comment by chinesedfan** [comment]
@pd4d10 Good except a typo. Then we can merge this one. If you can solve #868, that will be better.

File: src/components/issue-event-list-item.component.js (hunk 1 above)
Comment (pd4d10) [reply src/components/issue-event-list-item.component.js]: Thanks! code updated

**Pull request merged**